test/vonmises.jl

```diff
@@ -0,0 +1,36 @@
+# Tests for Von-Mises distribution
+
+using Distributions
+using Compat.Test
```

Comment:
```suggestion
using Test
```
Do we need legacy Test behavior here?

Reply:
@bdeonovic this is the last thing before merging I think